hooks/__tests__/useMediaQuery.test.tsx

```diff
@@ -0,0 +1,84 @@
+import { type FC } from 'react';
+import { render } from '@testing-library/react';
+import { useMediaQuery } from '../useMediaQuery';
+
+describe('useMediaQuery', () => {
+  const MediaQueryRenderer: FC = () => {
```

Comment:
Not an efficient way of testing a Hook. There are better methods to test a hook functionality 🙏 (no need to make a component)